operator/v1/types_network.go

```diff
@@ -533,6 +536,16 @@ type ProxyConfig struct {
 	ProxyArguments map[string]ProxyArgumentList `json:"proxyArguments,omitempty"`
 }
 
+// EgressIPConfig defines the configuration knobs for egressip
+type EgressIPConfig struct {
+	// ReachabilityTotalTimeout configures the EgressIP node reachability check timeout in seconds.
+	// When omitted, this means the user has no opinion and the platform is left to choose a reasonable default, which is subject to change over time.
+	// The current default is 1 second.
+	// A value of 0 disables the reachability check.
+	// +optional
+	ReachabilityTotalTimeoutSeconds *uint32 `json:"reachabilityTotalTimeoutSeconds,omitempty"`
```

Comment:
What does `Total` mean in this context?

Reply:
I tried to explain it above this more of number of retries with internal timeout and sleep such that it will add up to the configured value

Reply:
The context of why this is total should make its way into the godoc description please, this is helpful for end users when you use `oc explain`